consonance-common/src/main/java/io/consonance/arch/persistence/PostgreSQL.java

```diff
@@ -187,11 +188,10 @@ public Integer createProvision(Provision p) {
 
     public String createJob(Job j) {
         JSONObject jsonIni = new JSONObject(j.getIni());
-        JSONObject jsonExtraFiles = new JSONObject(j.getExtraFiles());
         Map<Object, Map<String, Object>> map = this.runInsertStatement(
-                "INSERT INTO job (status, job_uuid, workflow, workflow_version, job_hash, ini, extra_files) VALUES (?,?,?,?,?,?,?::jsonb)",
+                "INSERT INTO job (status, job_uuid, workflow, workflow_version, job_hash, ini) VALUES (?,?,?,?,?,?)",
```

Comment:
The postgres class is now only used by the daemons which update. At some point, I'll clean up this code and remove their ability to insert. I think Hibernate should create the job and associated extra-files column in one transaction.